internal/openshift/controller.go

```diff
@@ -31,7 +32,8 @@ type Controller interface {
 	GetUser(ns string) model.User
 }
 
-type OpenShiftController struct {
+// ControllerImpl watches OpenShift cluser for changes and implements Controller
+type ControllerImpl struct {
```

Comment:
Do you really need 'Impl'?

Reply:
I have to change it from OpenShiftController to something else. Can't keep `Controller`, because that the interface. Other name that I can think of is `OSController`, but `ControllerImpl` seems more appropriate to me.